vm/interpreter/src/vm.rs

```diff
@@ -198,6 +207,7 @@ where
         messages: &[BlockMessages],
         parent_epoch: ChainEpoch,
         epoch: ChainEpoch,
+        arc_store: std::sync::Arc<impl BlockStore + Send + Sync>,
```

Comment:
cant you get this from `self`?

Reply:
No AFAIR, i wasn't possible. Since vm stores DB by ref, I'd have to change it to `Arc<DB>` and that cascaded changes all the way to HAMT structures

Reply:
oh yeah gotcha